_posts/2016-06-14-https-securise-websites.md

```diff
@@ -0,0 +1,181 @@
+---
+layout: post
+title:  "How Etsy France switched all its sites to HTTPS"
+date:   2016-07-15
+categories: projects
+tags: [security, https, http, migration, project]
+author: antoine
+---
+
+When Etsy acquired A little Market, the migration of our sites to https which was on the shelf since a few years became one of our top priority projects in mid 2015.  
+But before starting such a big and impacting project, we went through a long preparation phase.
+
+## Project preparation
+
+### Why ?
+
+The goal of this project was to force all our users to access our services through HTTPS connections. Google and other search engines would have to crawl our websites only in HTTPS to avoid duplicate content.
```

Comment:
IMHO, it's not a why but a result. A why could have been to prevent session hijacking, or content tempering.